.editorconfig

```diff
@@ -16,10 +16,6 @@ indent_style = space
 indent_size = 2
 indent_style = space
 
-[Vagrantfile]
```

Comment:
I definitively don't miss the old Vagrant file :-)

Reply:
So, let's merge this? :)